neo4j-cli/aura/aura.go

```diff
@@ -29,6 +30,7 @@ func NewCmd(cfg *clicfg.Config) *cobra.Command {
 	if cfg.Aura.AuraBetaEnabled() {
 		cmd.AddCommand(dataapi.NewCmd(cfg))
 	}
+	cmd.AddCommand(_import.NewCmd(cfg))
```

Comment:
Are these supposed to be in production?

If this is using the beta APIs this should be behind the beta-enabled flag, like dataapi